docs/get-started/dapps/reach.md

```diff
@@ -0,0 +1,474 @@
+
+Now let’s get started. 
+
+# Organization
+
+This guide is organized into two sections. This document is the first section which helps you launch the dApp and run an auction simulation. The second section will provide a deeper dive into the different components of a reach application.
+
+
+All of the code for this guide is located [here] (https://github.com/algorand/reach-auction). The `index.mjs` and `index.rsh` files are included, so follow along!
+
+## Install Reach
+
+Reach is designed to work on POSIX systems with [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/) installed. The best way to install Docker on Mac and Windows is with [Docker Desktop](https://www.docker.com/products/docker-desktop). 
+
+
+To confirm everything is installed try to run the following three commands and see no errors.
+
+
+```
+  $ make --version
+  $ docker --version
+  $ docker-compose --version
+```
+
+
+If you’re using Windows, consult [the guide to using Reach on Windows](https://docs.reach.sh/guide-windows.html).
+
+Once confirmed that reach prerequisite are installed, choose a directory and clone the repo example so you can follow along:
+
+```
+  $ mkdir -p ~/reach/ && cd ~/reach/
+  $ git clone https://github.com/algorand/reach-auction
+```
+
+Navigate to the project folder
+
+``` bash
+cd reach-auction
+```
+
+Next, download Reach by running
+``` bash
+$ curl https://docs.reach.sh/reach -o reach ; chmod +x reach
+```
+
+Confirm the download worked by running
+
+
+``` bash
+$ ./reach version
+```
+
+
+Since Reach is Dockerized, when first used, the images it uses need to be downloaded. This will happen automatically when used for the first time, but can be done manually now by running
+
+
+```
+  $ ./reach update
+```
+
+
+You’ll know that everything is in order if you can run
+
+
+```
+  $ ./reach compile --help
+```
+
+
+To determine the current version is installed, run
+
+
+```
+  $ ./reach hashes
+```
+
+
+Output should look similar to:
+
+```
+reach: fb449c94
+reach-cli: fb449c94
+react-runner: fb449c94
+rpc-server: fb449c94
+runner: fb449c94
+devnet-algo: fb449c94
+devnet-cfx: fb449c94
+devnet-eth: fb449c94
+```
+
+
+All of the hashes listed should be the same and then visit the #releases channel on the [Reach Discord Server](https://discord.gg/9kbHPfwbwn) to see the current hashes.
+
+More information: Detailed Reach install instructions can be found in the [reach docs](https://reach.sh/). 
+
+
+
+## Setup environments and run tests
+
+
+Sometimes it may be convenient to use the reach run command, preceded by setting the **REACH_CONNECTOR_MODE **, especially when testing multiple blockchain deployments.
+
+```
+  REACH_CONNECTOR_MODE=ALGO-devnet ./reach run
+```
+
+
+Set an environment variable to use the Algorand Blockchain. 
+
+
+```
+export REACH_CONNECTOR_MODE="ALGO-devnet"
+```
+
+
+# Application Overview
```

Comment:
I like the overview section from PyTeal, as it has some visuals.

Reply:
added a visual